<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QColorWeight</class>
 <widget class="QWidget" name="QColorWeight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QColorWeight</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>720</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>灰度权重</string>
    </attribute>
    <widget class="QDockWidget" name="dockWidget_grayHitChart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>350</width>
       <height>200</height>
      </rect>
     </property>
     <property name="features">
      <set>QDockWidget::NoDockWidgetFeatures</set>
     </property>
     <property name="allowedAreas">
      <set>Qt::NoDockWidgetArea</set>
     </property>
     <property name="windowTitle">
      <string/>
     </property>
     <widget class="QWidget" name="dockWidgetContents"/>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>12</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>检测方法：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_selectMode">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_R">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>180</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>200</number>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_G">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>110</y>
       <width>180</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>200</number>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_B">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>150</y>
       <width>180</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_R">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>35</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_G">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>35</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_B">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>35</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_R">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>72</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_G">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>112</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_B">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>152</y>
       <width>31</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_grayLeft">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>210</y>
       <width>320</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_grayRight">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>220</y>
       <width>320</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_grayLeft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>250</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_grayRight">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>250</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_gray">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>250</y>
       <width>320</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>152</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>蓝(B)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>112</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>绿(G)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>72</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>红(R)</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>691</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_saveParams">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>320</y>
       <width>120</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>保存参数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_loadParams">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>320</y>
       <width>120</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>导入参数</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>彩色分布</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsView_grayColor">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_R"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_G"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_B"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_Gray"/>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R阀值</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_Rn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>308</y>
       <width>200</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>340</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>T阀值</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_Tn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>338</y>
       <width>200</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Tn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>338</y>
       <width>35</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Rn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>308</y>
       <width>35</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_grayImg">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_ColorImg">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_sourceImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
